src/haddock/modules/defaults.yaml

```diff
@@ -148,3 +148,5 @@ less_io:
     a network file system where I/O operations are slow.
   group: "execution"
   explevel: easy
+  incompatible:
```

Comment:
Nice!  Will the syntax allow for multiple entries under the incompatible category? I assume yes

Reply:
Default `yaml` format syntax applies here